frontend/packages/helm-plugin/src/components/forms/HelmChartRepository/CreateHelmChartRepositoryForm.tsx

```diff
@@ -51,17 +55,13 @@ const CreateHelmChartRepositoryForm: React.FC<FormikProps<FormikValues> & {
           ? ProjectHelmChartRepositoryModel
           : HelmChartRepositoryModel
       }
-      showSamples={!resource}
+      showSamples
```

Comment:
For other resources we show the samples only when creating a new resource, not in the edit flow.

Can we archive this by checking `existingRepo` here? Btw: Why `existingRepo` instead of `resource`, isn't it the same?

```suggestion
      showSamples={!existingRepo}
```